web/js/transcribe.js

```diff
@@ -2160,12 +2160,23 @@ function drawLinesDesignateColumns(lines, tool, RTL, shift, preview, restore) {
         console.warn("No lines found in a bad place...");
     }
     // we want automatic updating for the lines these texareas correspond to.
-    $("textarea")
+    $(".theText")
         .keydown(function(e) {
             //user has begun typing, clear the wait for an update
-            clearTimeout(typingTimer);
-            markLineUnsaved($(e.target).parent());
-
+            if (e.which === 13 && (tpen.screen.liveTool !=="parsing")){ //enter
```

Comment:
hmmm... I guess this is okay. It feels weird to have to do it this way, but I'm not too worried for now.